prestoadmin/server.py

```diff
@@ -782,6 +782,8 @@ def collect_node_information():
 
 
 def get_status_from_coordinator():
+    if check_presto_version() != '':
```

Comment:
This isn't quite right (this checks that presto is installed on your presto-admin node, and if not will print nothing out for the `server status` command). 

 But looking at this function again, I think it may be ok as it was.  Maybe see if the test_status tests fail from having the client raise an exception when it can't generate a config.  If it does I can give you more direction on how to rewrite it.